ios/scripts/kme-universal.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
```

Comment:
Can we standardise on `#!/usr/bin/env bash` for scripts into the future? (https://stackoverflow.com/a/16365367/1836776)

(and would recommend `brew install bash` on our macs.